grid-proxy/tools/db/utils.go

```diff
@@ -2,21 +2,32 @@ package main
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
 	"math/rand"
 	"net"
 	"reflect"
 	"strings"
 )
 
-func rnd(min, max uint64) uint64 {
-	return rand.Uint64()%(max-min+1) + min
+var errMaxMinInvalid = errors.New("min cannot be greater than max")
```

Comment:
we don't really need to create an error variable, right?
if it's to easily check it when testing, just checking if there is an error is usually enough.